tsconfig.json

```diff
@@ -5,8 +5,9 @@
     "target": "es5",
     "lib": ["es2017", "dom"],
     "outDir": "lib",
-    "rootDir": ".",
-    "jsx": "react"
+    "rootDirs": [".", "stories"],
```

Comment:
I wonder if we're starting to have enough in here that there should be one tsconfig for production and a separate one with tests and storybook?

Reply:
I tried it and it's possible but it doesn't change the compilation run time or the size of the production build so I'm not sure if doing this will be that helpful.